src/proxy.cc

```diff
@@ -1551,6 +1555,47 @@ void* ncclProxyService(void* _args) {
   return NULL;
 }
 
+void* ncclProxyServiceDaemon(void* _args) {
```

Comment:
Revise name to be related with resilience

Reply:
revised.